target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -76,6 +76,10 @@ mikrotik,sxtsq-5-ac)
 	ucidef_set_led_rssi "rssimediumhigh" "rssimediumhigh" "green:rssimediumhigh" "wlan0" "61" "100"
 	ucidef_set_led_rssi "rssihigh" "rssihigh" "green:rssihigh" "wlan0" "81" "100"
 	;;
+mikrotik,wap-ac)
+	ucidef_set_led_default "power" "POWER" "blue:power" "1"
+	ucidef_set_led_default "user" "USER" "green:user" "0"
```

Comment:
Default values can be set via DTS

Reply:
Dropped this. The entries won’t show up in `/etc/config/system` anymore, but I suppose that’s fine.

Reply:
They are still controllable like any other LED

Reply:
I typically disagree with this choice: IMO the user led should be ON after a successful boot. That's the default for nearly all openwrt devices, and the de-facto default on other mikrotik platforms. "solid led" == "system running normally". The power led cannot replace this information as a device can be powered and yet not running correctly.

Reply:
I can switch it back if preferred. My rationale is that I was trying to:
 - match the status quo before I removed the `ucidef_set_led_default` which created an `/etc/config/system` entry that turned the LED off.
 - match the most similar device, mikrotik,cap-ac, in behavior if not implementation. It also matches compex,wpj428.
 - disambiguate states. The bootloader makes use of the user LED as well. For devices that have a separate power LED, I was think of the user/diag LED as something that’s on at various blink states during boot, and then turns off when the boot is complete. This device does have a separate power LED. (My thoughts here may be “colored” by my long experience with the ath79/netgear,wndr3700 family.)

Fine by me to do it either way. I don’t have a strong opinion and am happy to go with the consensus on this point, and I can do a follow-up (untested) to fix mikrotik,cap-ac and even compex,wpj428 afterwards if needed. Other thoughts?

Reply:
Well the cap-ac does define led-running and then turns if off, the latter action is wrong IMO and should be fixed. *All* other mikrotik device in this target (hap-ac*, lhg60...) define led-running and leave it on, which also match common openwrt behavior (and other mikrotik devices in other targets). I'm not familiar with the compex,wpj428 so I can't comment there, but my recommendation for consistency's sake would be to:

- define led-running
- fix cap-ac to bring it in line with other devices

As I explained, the power LED is too ambiguous to report a complete and successful boot, that should be the purpose of the status (user/diag) led. AIUI the main reason for giving control over the power led (when it's available) is for e.g. users who want all LEDs off in their particular setup. But that is obviously not the default behavior nor a desirable default for most users :)

HTH

Reply:
I updated this to restore `green:user` as `led-running`.

I can handle reconciling mikrotik,cap-ac in a follow-up. The diag LED and the default network configuration, at least, should be kept consistent.